pkg/provider/azure_vmss.go

```diff
@@ -825,7 +825,15 @@ func (ss *ScaleSet) listScaleSetVMs(scaleSetName, resourceGroup string) ([]*armc
 	ctx, cancel := getContextWithCancel()
 	defer cancel()
 
-	allVMs, rerr := ss.ComputeClientFactory.GetVirtualMachineScaleSetVMClient().ListVMInstanceView(ctx, resourceGroup, scaleSetName)
+	var allVMs []*armcompute.VirtualMachineScaleSetVM
+	var rerr error
+	if ss.Config.VmssVirtualMachineCacheWithoutInstanceView {
+		klog.V(4).Info("listScaleSetVMs called for scaleSetName: ", scaleSetName, " resourceGroup: ", resourceGroup)
+		allVMs, rerr = ss.ComputeClientFactory.GetVirtualMachineScaleSetVMClient().List(ctx, resourceGroup, scaleSetName)
+	} else {
+		klog.V(4).Info("listScaleSetVMs called for scaleSetName with instanceView: ", scaleSetName, " resourceGroup: ", resourceGroup)
```

Comment:
set as V(6) to avoid log flushing since this is a frequent operation

Reply:
Done